lib/views/invoice_slack_view.ex

```diff
@@ -0,0 +1,29 @@
+defmodule Aprb.Views.InvoiceSlackView do
+  import Aprb.ViewHelper
+
+  def render(event) do
+    partner_data = fetch_partner_data(event["properties"]["partner_id"])
+    %{
+      text: ":money_with_wings: Invoice #{event["object"]["display"]} #{event["verb"]}",
+      attachments: "[{
```

Comment:
I remember trying that originally and had issues with it! but im totally down for trying it again and clean up this string mess hehehe.